es6/docxtemplater.js

```diff
@@ -28,15 +28,22 @@ const {
 const currentModuleApiVersion = [3, 21, 0];
 
 const Docxtemplater = class Docxtemplater {
-	constructor() {
-		if (arguments.length > 0) {
+	constructor(zip, { modules = [] } = {}) {
+		if (!Array.isArray(modules)) {
 			throw new Error(
-				"The constructor with parameters has been removed in docxtemplater 3, please check the upgrade guide."
+				"Please pass modules as an array"
```

Comment:
Can you change the error for the modules : 

throw new Error("The modules argument of docxtemplater's constructor must be an array");

Reply:
Yes, that looks more readable error. I will do it. 👍